### PR TITLE
block account transfer on non magi referral

### PR DESCRIPTION
**Pull request by jayreddy519**
[RM-99035](https://redmine.dchbx.org/issues/99035)

### CODE REVIEW

File: config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml [hunk 1]
```diff
@@ -138,10 +138,8 @@ registry:
     - key: :transfer_service
       is_enabled: <%= ENV['TRANSFER_SERVICE_IS_ENABLED'] || true %>
       item: "aces"
-    - key: :non_magi_medicaid_eligible
-      is_enabled: <%= ENV['NON_MAGI_MEDICAID_ELIGIBLE_IS_ENABLED'] || false %>
-    - key: :eligible_for_non_magi_reasons
-      is_enabled: <%= ENV['ELIGIBLE_FOR_NON_MAGI_REASONS_IS_ENABLED'] || false %>
+    - key: :non_magi_transfer
+      is_enabled: <%= ENV['NON_MAGI_TRANSFER_IS_ENABLED'] || false %>
```
Comment (mdkaraman) [inline config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml:142]: We should use a default value of 'true' for ME config of non_magi_transfer feature in order to preserve the current behavior for that client.

**Review by mdkaraman — DISMISSED** [review]

File: config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml [hunk 2]
```diff
@@ -138,10 +138,8 @@ registry:
     - key: :transfer_service
       is_enabled: <%= ENV['TRANSFER_SERVICE_IS_ENABLED'] || true %>
       item: "aces"
-    - key: :non_magi_medicaid_eligible
-      is_enabled: <%= ENV['NON_MAGI_MEDICAID_ELIGIBLE_IS_ENABLED'] || false %>
-    - key: :eligible_for_non_magi_reasons
-      is_enabled: <%= ENV['ELIGIBLE_FOR_NON_MAGI_REASONS_IS_ENABLED'] || false %>
+    - key: :non_magi_transfer
+      is_enabled: <%= ENV['NON_MAGI_TRANSFER_IS_ENABLED'] || true %>
```
Comment (markgoho) [inline config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml:142]: both of these fallbacks were false, why are we changing this one to `true`?

Comment (jayreddy519) [reply config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml:142]: i have changed the code related to this mec check, as per the code change, to preserve the current functionality we need to set this to TRUE for ME.

Comment (markgoho) [reply config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml:142]: @jayreddy519 this is not where we set the value -- this is the fallback. Please change it to `false`

**Review by markgoho — APPROVED** [review]

**Pull request merged**